docs/codeql/codeql-cli/codeql-cli-reference.rst

```diff
@@ -9,12 +9,14 @@ Learn more about the files you can use when running CodeQL processes and the res
    :titlesonly:
    :hidden:
 
+   about-codeql-packs
    about-ql-packs
    query-reference-files
    sarif-output
    exit-codes
 
 
+- :doc:`About CodeQL packs <about-codeql-packs>`: CodeQL packs are created with the CodeQL package manager and are used to create, depend on, publish, and run CodeQL queries and libraries.
```

Comment:
```suggestion
- :doc:`About CodeQL packs <about-codeql-packs>`: CodeQL packs are created with the CodeQL CLI and are used to create, depend on, publish, and run CodeQL queries and libraries.
```
Not sure if my suggestion is the best wording, but it might be a little confusing to mention a "CodeQL package manager" when it's really just a set of new commands in the existing CLI.
It might also be sensible to say "...created with the `codeql pack create`" command..."

Reply:
Great point. Since this is just a brief introduction, I think your suggestion works well.